### ISSUE TITLE
Missing capability for slice-to-volume correction using 'eddy_slspec' config parameter 

### ISSUE THREAD

**Issue by bacaron** [brain-life/app-mrtrix3-preproc#19]
Hello,

I'm running into an issue on the 1.7 branch where slice-to-volume correction isn't being performed (causing the app to fail) due to a missing -eddy_slspec flag. This happens even though I've inputted the appropriate info into the 'eddy_slspec' config parameter.

I remember this working in the past, so I think the functionality is somewhere in the code, but maybe just commented out?

I've attached the full error log file
[slurm-1186202.err.txt](https://github.com/brainlife/app-mrtrix3-preproc/files/6480495/slurm-1186202.err.txt)

Any help would be greatly appreciated!

Thanks

**Comment by soichih** [comment]
I am not sure why the code to handle eddy_slspsec is commented out for 1.7, but it should work with master branch. @bcmcpher Should we cut a new release version using the current master branch?

**Comment by bcmcpher** [comment]
Yeah, master should be ready for a new release w/ some fixes that ought to deal with this ok
